mean calculator/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mean Calculator</title>
+</head>
+<body>
+    <script>
+        var nums = [1, 2, 3, 4, 5];
+
+        meanCalculator(nums)
+
+        function meanCalculator(){
+
+                var totalSum = 0;
+                
+                for(var i in nums){
+                    totalSum += nums[i];
+                }
+
+                var numsCnt = nums.length;
+
+                var average = totalSum / numsCnt;
+
+                console.log('The mean is: ' + average);
```

Comment:
Make sure you are using a return statement instead of a console log. Console logs should only be used for debugging purposes.

Reply:
Understood, thank you. I'll update it